paasta_tools/cli/cmds/local_run.py

```diff
@@ -842,7 +856,7 @@ def run_docker_container(
     else:
         chosen_port = pick_random_port(service)
     environment = instance_config.get_env_dictionary()
-    secret_volumes = {}
+    secret_volumes = {}  # type: ignore
```

Comment:
Ignoring type here because it would requiring either changing how we write to file (as `str` first and  then as `bytes`) or changing signature of https://github.com/Yelp/paasta/blob/8133eaa2c062c918c2efd7ec32a78d0f5f796a2f/paasta_tools/secret_tools.py#L196